BDC/src/data_collector.py

```diff
@@ -0,0 +1,62 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2023 Ruchita Nathani <ruchita.nathani@fau.de>
+import csv
+import requests
+import json
+
+
+class DataCollector:
+    def __init__(self):
+        self.data = []
+
+    @staticmethod
+    def get_data_from_csv():
+        """Retrieve information from the CSV file and utilize it in the Google API"""
+        data = []
+        with open('../data/given_data.csv', 'r', encoding='utf8') as file:
```

Comment:
This relative path will result in a FileNotFoundError if it's run from anywhere else than inside `/BDC/src/`. I suggest using absolute paths:  
```suggestion
        file_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), '../data/given_data.csv')
        with open(file_path, 'r', encoding='utf8') as file:
```